gradle.properties

```diff
@@ -14,6 +14,7 @@ lincheck_version=2.18.1
 dokka_version=1.9.20
 byte_buddy_version=1.10.9
 reactor_version=3.4.1
+reactor_docs_version=3.4.5
```

Comment:
Shouldn't we update reactor version instead?

Reply:
I thought we were trying to depend on the earliest version possible, so as not to force the users to upgrade to the new Reactor just because of our integration. There are some repositories with 3.3 and 3.4 versions of Reactor mentioned as dependencies.